gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -103,6 +103,11 @@ message FeedEntity {
   optional TripUpdate trip_update = 3;
   optional VehiclePosition vehicle = 4;
   optional Alert alert = 5;
+  optional Trip trip = 6;
+  optional Shape shape = 7;
+  optional Route route = 8;
+  optional Stop stop = 9;
+  optional StopTime stopTime = 10;
```

Comment:
The variable names are snake-cased afaics, so `stopTime` should be `stop_time` likely (like for `trip_update`).

Reply:
I went ahead and edited accordingly in https://github.com/MobilityData/transit/pull/47/commits/536a9932a6825f30c22ae8d4216eca1a1b09ca75 in this PR so I can regenerate the bindings.